docs/source/configuration/multinode_cassandra.rst

```diff
@@ -0,0 +1,112 @@
+============================
+
+The following behaviour is guaranteed when interacting with a Cassandra cluster with three nodes, RF three, and one node down. More generally, this is the expected behaviour for a cluster that has at least one but fewer than RF / 2 nodes down.
+
+.. list-table::
+    :widths: 40 40
+    :header-rows: 1
+
+    *    - Cassandra KVS API Method
+         - Behaviour
+
+    *    - ``addGarbageCollectionSentinelValues``
+         - Same as when all nodes are up.
+
+    *    - ``cleanUpSchemaMutationLockTablesState``
+         - Same as when all nodes are up.
+
+    *    - ``close``
+         - Same as when all nodes are up.
+
+    *    - ``compactInternally``
+         - Same as when all nodes are up.
+
+    *    - ``create``
+         - Same as when all nodes are up.
+
+    *    - ``createTable``
+         - Throws ``IllegalStateException``
+
+    *    - ``createTables``
+         - Throws ``IllegalStateException``
+
+    *    - ``delete``
+         - Throws ``PalantirRuntimeException``
+
+    *    - ``dropTable``
+         - Throws ``IllegalStateException``
+
+    *    - ``dropTables``
+         - Throws ``IllegalStateException``
+
+    *    - ``get``
+         - Same as when all nodes are up.
+
+    *    - ``getAllTableNames``
+         - Same as when all nodes are up.
+
+    *    - ``getAllTimestamps``
+         - Throws ``PalantirRuntimeException``
+
+    *    - ``getLatestTimestamps``
+         - Same as when all nodes are up.
+
+    *    - ``getMetadataForTable``
+         - Same as when all nodes are up.
+
+    *    - ``getMetadataForTables``
+         - Same as when all nodes are up.
+
+    *    - ``getRange``
+         - Same as when all nodes are up.
+
+    *    - ``getRangeOfTimestamps``
+         - Throws ``InsufficientConsistencyException`` (`when the iterator is accessed`).
+
+    *    - ``getRows``
+         - Same as when all nodes are up.
+
+    *    - ``getRowsColumnRange``
+         - Same as when all nodes are up.
+
+    *    - ``multiPut``
+         - Same as when all nodes are up.
+
+    *    - ``put``
+         - Same as when all nodes are up.
+
+    *    - ``putMetadataForTable``
+         - Throws ``IllegalStateException`` after 1 minute.
+
+    *    - ``putMetadataForTables``
+         - Throws ``IllegalStateException`` after 1 minute.
+
+    *    - ``putUnlessExists``
+         - Same as when all nodes are up.
+
+    *    - ``putWithTimestamps``
+         - Same as when all nodes are up.
+
+    *    - ``truncateTable``
+         - Throws ``PalantirRuntimeException``
+
+    *    - ``truncateTables``
+         - Throws ``PalantirRuntimeException``
+
+Less than a quorum of nodes are up
+==================================
+
+Superuser authentication will fail with a ``AuthenticationException``, since quorum is necessary for this operation. A non-super user authentication may still succeed, but all of the above operations should fail. `Note that this behaviour is still being verified!`
```

Comment:
I'm confused by "the behaviour is still being verified". Are we not sure that this is the case?